chat/public/js/src/chat.js

```diff
@@ -361,9 +361,8 @@ function ChatXBlock(runtime, element, init_data) {
      * pause: delays execution with a timeout
      */
     var pause = function(timeout) {
-        return new Promise(function(resolve, reject) {
-            setTimeout(resolve, timeout);
-        });
+        var promise = $.Deferred();
+        setTimeout(promise.resolve, timeout);
```

Comment:
@mtyaka Brilliant! It did, at least locally - let's see what Circle says. That saved me a big bug hunt 👍